src/styles-new/sass-base.scss

```diff
@@ -25,8 +25,8 @@
 // Do not include anything that renders to CSS!
 @import "constants";
 @import "constants-mobile.scss";
-//@import "constants-espresso"; // TEMP
-@import "constants-snow"; // TEMP
+@import "constants-espresso"; // TEMP
+//@import "constants-snow"; // TEMP
```

Comment:
Can we keep the default snow for now? It's just one less thing to remember with VISTA deployments...

Reply:
@akhenry Thanks for the review! Changed the default back to snow.